src/pages/components/tabs-extended.mdx

```diff
@@ -49,4 +51,32 @@ On mobile, Tabs extended switches to an accordion for easier browsing.
 </Column>
 </Row>
 
+## Options
+
+Tabs extended media is a variation that constructs its content using [Content item horizontal](/components/content-item-horizontal) with media to create an organized section optimized for content with media. This is useful especially when reducing page length is desired. Keep in mind some content will be hidden until user interaction, so they should not be critical information. 
+
+For more on Tabs extended media, you can view the [functional specs](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Tabs-extended-media) and [Web Components Storybook](https://www.ibm.com/standards/carbon/web-components/?path=/story/components-tabs-extended-with-media--default).
+
+<Row>
+<Column colMd={8} colLg={12}>
+
+![Image of Tabs extended with media component](../../images/component/tabs-extended/components--tabs-extended-media-1.jpg)
+
+
+</Column>
+</Row>
+
+### Behavior
```

Comment:
@RichKummer I don't think you need a new section here for behavior because it's already covered above. You could just show the two images but add captions underneath each one. 

Tabs extended with the media option, on desktop

Tabs extended with the media option, on mobile